Dockerfile

```diff
@@ -10,9 +10,12 @@ RUN pip3 install --no-cache-dir colorlog cython
 
 # For the nmap tracker, bluetooth tracker, Z-Wave
 RUN apt-get update && \
-    apt-get install -y --no-install-recommends nmap net-tools cython3 libudev-dev sudo libglib2.0-dev && \
+    apt-get install -y --no-install-recommends nmap net-tools cython3 libudev-dev sudo libglib2.0-dev bluetooth libbluetooth-dev && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Install pybluez for bluetooth tracker
+RUN pip install pybluez
```

Comment:
Actually, this can cause issues when the latest version that pip installs is different from what the bluetooth tracker expects. So I would like to suggest that we do not install it. It will be automatically when a user uses it - and since the correct packages have been installed, the installation won't fail.